ui/pages/signup/index.js

```diff
@@ -148,8 +165,64 @@ export default function Signup() {
             <ErrorMessage message={errors.confirmPassword} />
           )}
         </div>
+        <div className='my-2 w-full pt-6 text-2xs uppercase leading-none text-gray-400'>
+          Organization
+        </div>
+        <div className='my-2 w-full'>
+          <label htmlFor='orgName' className='text-3xs uppercase text-gray-500'>
+            Name
+          </label>
+          <input
+            required
+            id='orgName'
+            placeholder='name your organization'
+            onChange={e => {
+              setOrgName(e.target.value)
+              setErrors({})
+              setError('')
+              if (automaticOrgDomain) {
+                setSubDomain(getURLSafeDomain(e.target.value))
+              }
+            }}
+            className={`mb-1 w-full border-b border-gray-800 bg-transparent px-px py-2 text-2xs placeholder:italic focus:border-b focus:outline-none focus:ring-gray-200 ${
+              errors.orgName ? 'border-pink-500/60' : ''
+            }`}
+          />
+          {errors.orgName && <ErrorMessage message={errors.orgName} />}
+        </div>
+        <div className='my-2 w-full'>
+          <label
+            htmlFor='orgDoman'
+            className='text-3xs uppercase text-gray-500'
+          >
+            Domain
```

Comment:
I'd like to put a tooltip here that explains what the domain is for, but that can be a separate change.